plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -52,7 +52,8 @@ event BlockSubmitted(
 
  > ***Note**: For more information about parameter types, return types, valid values, and so on, see the [Plasma Contract API documentation](../contracts/BlockController.md#submitblock).*
  
-
+# Block retrieval
+To learn what blocks need to be retrieved during sync, read `PlasmaFramework.nextChildBlock()` and `PlasmaFramework.childBlockInterval()`. Compute the block numbers by iterating every `childBlockInterval` from `0 + childBlockInterval` up to `nextChildBlock - childBlockInterval`, inclusive. Next, read the Merkle tree root of each block by calling `PlasmaFramework.blocks(blknum)`.
```

Comment:
`...Compute the block numbers by iterating` isn't very clear for me. I guess we need to put some math at it.
Block numbers are elements of arithmetic progression with a common difference of `childBlockInterval`, starting from ...